queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name like = '%mon';
+SELECT * from animals WHERE date_of_birth BETWEEN DATE '2016-01-01' and '2019-12-31'; 
```

Comment:
- [ ] Kindly note that the requirements of this project say that this query should only select names of all animals born between 2016 and 2019. only names not all columns.
![image](https://user-images.githubusercontent.com/65068771/178379690-0f71ba7f-bad9-449c-a01b-564f6948917d.png)